tests/test_project/tests/test_qa.py

```diff
@@ -184,6 +198,9 @@ def test_qa_call_check_commit_message_failure(self):
             with patch('argparse._sys.argv', option):
                 try:
                     check_commit_message()
+                    message = 'must be renamed to something more descriptive'
+                    self.assertContains(io.StringIO().getvalue(),
+                                        message)
```

Comment:
Umm.. why would commitcheck say "must be renamed to something more descriptive"? 
Also, the same questions as above? :smile:

Reply:
@atb00ker It checks if failure message is printing.
tests these lines:
https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L55
https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L158

Reply:
By "same questions as above, I meant:
- Is this affecting the coverage?
- Also, can this be moved to else clause?

Reply:
Okay but read the message text. The commit text would not include "must be renamed to something more descriptive", right?

Reply:
> Is this affecting the coverage?

yes it is, it checks two lines which weren't tested before

> Also, can this be moved to else clause?

yes, I can create 2 new functions instead

> Okay but read the message text. The commit error message would not include "must be renamed to something more descriptive", right?

according to https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L33 it should, right?

Reply:
> Also, can this be moved to else clause?

yes, I will move it in few hours

Reply:
> according to https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L33 it should, right?

but that's check migration name, I am highlighting the test called
"test_qa_call_check_commit_message_failure"
Why will commit_check test run migration_check, or am I missing something? :sweat_smile:

Reply:
Oh, you are right, commit check should be totally different, fixed. thanks